react/features/base/ui/components/updateTheme.native.ts

```diff
@@ -5,5 +5,8 @@
  * @returns {Object}
  */
 export default function updateTheme(theme: Object) {
+    //Gracetech -- to compact more buttons 
+    theme.spacing[2] = 2;
```

Comment:
this is very hacky, is there a better way?

Reply:
fixed